.env.sample

```diff
@@ -0,0 +1,9 @@
+TWITTER_API_KEY=
+TWITTER_API_SECRET=
+TWITTER_ACCESS_TOKEN=
+TWITTER_ACCESS_TOKEN_SECRET=
+
+NODE_ENV=development
+LOG_LEVEL=info
+PRETTY_PRINT=true
+PORT=3000
```

Comment:
Nitpick but I'd configure your IDE to include line breaks automatically at the end of files, I think the linting has caught it for any of your JS files but any of the config related pieces are missing it.

Reply:
what can we use to do this? editorconfig perhaps? we should include it in the bench template if so

Reply:
editorconfig would do the job here definitely, but from a quick look it would require people to install the extension as it's not included by default.

https://github.com/microsoft/vscode/issues/95902

Reply:
Fixed this specific case - but good point 👍